openapi-cli/src/test/resources/ballerina-to-openapi/query/query_scenario09.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/http;
```

Comment:
license header

Reply:
We do not have it in any other test files. @lnash94 should we add it to all the test files?

Reply:
FTM, we can avoid adding license headers to the test file since any other test files don't include the header. In future let's refactor the whole tests with headers.